sound/soc/sof/intel/hda-loader.c

```diff
@@ -329,25 +337,25 @@ int hda_dsp_cl_boot_firmware(struct snd_sof_dev *sdev)
 
 	/* try ROM init a few times before giving up */
 	for (i = 0; i < HDA_FW_BOOT_ATTEMPTS; i++) {
+		dev_dbg(sdev->dev,
+			"Attempting iteration %d of Core En/ROM load...\n", i);
```

Comment:
nit-pick: I think the original way of reporting *here* was a bit better: only `dev_dbg()` if an iteration fails, if the first iteration already succeeds, no need to print anything?

Reply:
@lyakh @kv2019i I considered both your suggestions but unfortunately our CI scripts looks for words like error, failed, panic etc and even if I remove the word error, I'd still have to reword it to remove the word failed.